generator/index.html

```diff
@@ -343,7 +345,15 @@ <h2 id="sprite-name"></h2>
   input.addEventListener('input', function(e) {
     console.log(input.value);
     var id = id_from_url(input.value);
-    input.value = "http://scratch.mit.edu/projects/" + id;
+    if(id) {
+      if(input.value.indexOf("phosphorus") > input.value.indexOf("scratch")) {
+        input.value = "http://phosphorus.github.io/#" + id;
```

Comment:
We should probably show a Scratch link in the input, regardless of where the source link came from. Thoughts?